### DISCUSSION TITLE
blink.cmp as nvim-cmp’s alternative

### DISCUSSION THREAD

**Discussion by 0xJWLabs** [LazyVim/LazyVim#4679] [Extra Requests]
I’d like to propose adding [blink.cmp](https://github.com/Saghen/blink.cmp) as an alternative completion engine in LazyVim’s extras section. This plugin has shown significant performance improvements over nvim-cmp, making it an excellent option for users seeking faster completion, especially on larger codebases or lower-performance machines.

## Justification

	1.	Performance Benefits: blink.cmp is optimized for speed and can provide a much faster response time than nvim-cmp, which is beneficial in environments where completion speed is critical.
	2.	Flexibility for Users: Including blink.cmp in the extras would allow users to easily switch between completion engines based on their preference and performance needs.

**Comment by folke** [comment]
See https://github.com/LazyVim/LazyVim/pull/4680
